docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,90 @@
+# Release 381 (11 May 2022)
+
+## General
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+* Fix failure when reading timestamp values with `row` type that are not ISO
```

Comment:
@ebyhr here is the reworded entry .. please confirm

Reply:
Looks good to me. Thank you.

Reply:
This is not very clear. Timestamp values cannot be of row type. Do you mean "nested timestamp values"?